pkg/store/prometheus.go

```diff
@@ -488,14 +489,9 @@ func (p *PrometheusStore) startPromRemoteRead(ctx context.Context, q *prompb.Que
 
 // matchesExternalLabels returns false if given matchers are not matching external labels.
 // If true, matchesExternalLabels also returns Prometheus matchers without those matching external labels.
-func matchesExternalLabels(ms []storepb.LabelMatcher, externalLabels labels.Labels) (bool, []*labels.Matcher, error) {
```

Comment:
ditto, not change the interface?

Reply:
this way, it has less signature to modify

Reply:
done, thanks